modules/storage/storage.go

```diff
@@ -0,0 +1,287 @@
+// Copyright 2014 The Gogs Authors. All rights reserved.
```

Comment:
As above.

Reply:
@lunny there's `// Copyright 2020 The Gitea Authors. All rights reserved.` below this line, do you mean just remove this one?

Reply:
Fixed